pkg/updates/updates.go

```diff
@@ -0,0 +1,79 @@
+package updates
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/google/uuid"
+	"github.com/redhatinsights/edge-api/pkg/devices"
+)
+
+// MakeRouter adds support for operations on update
+func MakeRouter(sub chi.Router) {
+	sub.Get("/", getDevices)
+	sub.Route("/device_uuid={device_uuid}", func(r chi.Router) {
```

Comment:
I could be wrong but I would think this would just be:

```suggestion
	sub.Route("/{device_uuid}", func(r chi.Router) {
```